frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -52,43 +57,77 @@ function renderReports(reports) {
       </Tag>
     )) : '';
 
+    const history = useHistory();
     const idKey = `my_alerts_${id}`;
     const idLink = `/activity-reports/${id}`;
+    const contextMenuLabel = `View activity report ${id}`;
     const statusClassName = `smart-hub--table-tag-status smart-hub--status-${status}`;
 
+    const { isOpen, openModal, closeModal } = useModal();
+    const onDelete = (reportId) => {
+      deleteReport(reportId);
+      fetchReports();
```

Comment:
For some reason refetchting the reports isn't working, you have to manually refresh to see the report no longer there. Looking into it.

Reply:
Looks like the DB isn't being updated fast enough for when the a new query is being searched for? They data that is returned when re-fetching the db gives a status of DRAFT for the deleted report, but when checking the db manually after, it's status is set to DELETED. Deployed to staging to check if this happens in deployment. Looks like accounts have been reset in staging tho